src/components/FormStep.js

```diff
@@ -339,16 +451,20 @@ const FormStep = ({form, submission, onLogicChecked, onStepSubmitted, onLogout})
 
     // now the actual checking *can* be aborted, which results in an exception being thrown.
     try {
-      // call the backend to do the check
+      // `checkAbortedLogicCheck` throws error if checko should be aborted, skipping `doLogicCheck`.
       checkAbortedLogicCheck(controller.signal);
-      // update our view of the data, which may have been changed by now because of user input
+
```

Comment:
a bunch of these newlines break up the code more, and hurt readability in the sense that the entire function no longer fits in the editor window to try to grok it. Again - please refrain from making code style updates for the sake of updating the code style.